js-intro.js

```diff
@@ -4,14 +4,31 @@
 // ---------------------------------------------
 // Consider this variable:
 
-var mantra = "Be the dev"
-
+// var mantra = "Be the dev"
 // 1a. Write the code that returns 'B' from mantra.
 
+var mantra = "Be the dev"
+var letter = mantra.charAt(0)
+console.log(letter)
+
 // 1b. Write the code that determines if there is an 'x' in mantra.
 
+function determine(){
+    var mantra = "Be the dev";
+    var lookFor = mantra.search("x");
+    return mantra.search("x") + " means the letter is not present"
+}
+console.log(determine("x"))
+
```

Comment:
If you want to pass the x into your function, you have to set up an argument. Right now the function will only return "-1 means the letter is not present"